Client/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Test and lint job on GitHub Actions
 - Mobile homepage
 - Histoire setup
+- Desktop homepage
+- Login and Register modal
```

Comment:
Changes in changelog must have the task tag like - [API-42] before the description